### PR TITLE
MODKBEKBJ-349 Set access type on create custom package

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
On the Create Custom package assign it to access type if it exists in POST request 

## Approach
Created a new DB table where packages and access types are linking

### CODE REVIEW

**Comment by psmagin** [comment]
Please review this and tell me if I am on the correct way to implement it. 
I will create tests after your review.

File: src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java [hunk 1]
```diff
@@ -186,22 +189,55 @@ public void getEholdingsPackages(String filterCustom, String q, String filterSel
...
+      template.requestAction(context ->
+        accessTypesService.existsById(accessTypeId, okapiHeaders)
+          .thenCompose(aBoolean -> {
+            if (Boolean.TRUE.equals(aBoolean)) {
+              return getVendorId(context)
+                .thenCompose(id -> context.getPackagesService().postPackage(packagePost, id))
+                .thenApply(packageById -> new PackageResult(packageById, null, null))
+                .thenCompose(packageResult -> updateAccessType(accessTypeId, packageResult, okapiHeaders));
+            } else {
+              throw new ValidationException("Access type does not exist");
```
Comment (natali-zaitseva) [inline src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java]: Could you use InputValidationException instead of ValidationException, please?

Comment (natali-zaitseva) [reply src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java]: and use constant for an error message like ACCESS_TYPE_NOT_FOUND_MESSAGE as it probably will be used in the future PUT method

Comment (psmagin) [reply src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java]: Fixed.

**Comment by psmagin** [comment]
Do we need to set up the access type in the `relationships` of the package as per jsonapi?

**Pull request merged**